book/src/generated/typable-cmd.md

```diff
@@ -41,3 +41,5 @@
 | `:hsplit`, `:hs`, `:sp` | Open the file in a horizontal split. |
 | `:tutor` | Open the tutorial. |
 | `:goto`, `:g` | Go to line number. |
+| `:sort` | Sort lines in selection. |
```

Comment:
This should be updated to say that it sorts ranges in the selection now. Might even be worth putting an explicit note that it does *not* simply sort the lines.

Reply:
Updated the docs. Kept it without the explicit note for now otherwise the description was very long. We can add it in the future if we have `help` for commands.